tests/text/test_rouge.py

```diff
@@ -48,16 +49,57 @@
     "targets": [["The quick brown dog jumps on the log."], ["Is your name John"]],
 }
 
+BATCHES_3 = {
```

Comment:
can we move the test inputs to single file #666

Reply:
Can be done. Should I go ahead with it ?

Reply:
yes, please

Reply:
Done. Please check